src/field/richtext/RichTextToolbar.tsx

```diff
@@ -164,6 +166,67 @@ export const RichTextToolbar = forwardRef(function RichTextToolbar(
               </DropdownMenu.Item>
             </DropdownMenu.Items>
           </DropdownMenu.Root>
+          <DropdownMenu.Root top>

```

Comment:
Can we make this optional by adding an `enableTables?: boolean` to [`RichTextOptions`](https://github.com/alineacms/alinea/blob/d65a1dcdd8a2b0635ed11bd22a1480f436e22a09/src/field/richtext/RichTextField.tsx#L8)?

Reply:
[(https://github.com/alineacms/alinea/pull/381/commits/cb1e41f5b99087898f8c51145b3ec4ff4db1fb09)]